bundle/Resources/config/role.yml

```diff
@@ -95,9 +105,71 @@ services:
         tags:
             - { name: ez.limitation.formMapper, limitationType: Class }
 
+    ezrepoforms.limitation.form_mapper.parentcontenttype:
+        parent: ezrepoforms.limitation.form_mapper.contenttype
+        class: %ezrepoforms.limitation.form_mapper.parentcontenttype.class%
+        arguments: [@ezpublish.api.service.content_type]
```

Comment:
This is not needed as you already defined `ezrepoforms.limitation.form_mapper.contenttype` as parent.